### PR TITLE
Invite accept method

### PR DESCRIPTION
**Pull request by Vap0r1ze**

### CODE REVIEW

File: src/client/rest/RESTMethods.js [hunk 1]
```diff
@@ -495,6 +495,14 @@ class RESTMethods {
     });
   }
 
+  acceptInvite(invite) {
+    return new Promise((resolve, reject) => {
+      this.rest.makeRequest('post', Contants.Endpoints.invite(invite.code), true)
```
Comment (Tyler0130) [inline src/client/rest/RESTMethods.js]: Should be "Constants" instead of "Contants".

Comment (Tyler0130) [inline src/client/rest/RESTMethods.js]: Should be "Constants" instead of "Contants".

**Comment by iCrawl** [comment]
Make sure you check why the built failed.
ESLint threw 2 errors.

**Comment by Tyler0130** [comment]
Change the branch to `indev` instead of `master`.

**Comment by Vap0r1ze** [comment]
nevermind

**Pull request closed without merging**